nupic/research/frameworks/vernon/handlers.py

```diff
@@ -1059,14 +1056,277 @@ def get_execution_order(cls):
         return eo
 
 
+class MetaContinualLearningExperiment(SupervisedExperiment):
+
+    def setup_experiment(self, config):
+
+        # Simplify loading
+        config["model_args"]["num_classes"] = config["num_classes"]
```

Comment:
This requires the model to accept num_classes as an arg. Try
```
if "num_classes" in model_args:
     num_classes = config["model_args"]["num_classes"]
     config.setdefault("num_classes", num_classes)
```

Reply:
Won't `num_classes` be ignored by the model if it's not required for initialization?

Reply:
It will be called as `model_class(**model_args)`, which unpacks all args whether or not `model_class` accepts them.

Reply:
Fixed.